### PR TITLE
fix windows aarch64 bundle not found

### PR DESCRIPTION
**Pull request by 1595901624**
image

When building windows-aarch64, looking for x64 bundle.

### CODE REVIEW

**Comment by FabianLars** [comment]
Nice, thank you! Could you also add a changefile similar to this one please? https://github.com/tauri-apps/tauri-action/blob/dev/.changes/fix-tauri-init.md

**Comment by 1595901624** [comment]
> Nice, thank you! Could you also add a changefile similar to this one please? https://github.com/tauri-apps/tauri-action/blob/dev/.changes/fix-tauri-init.md很好，谢谢！您能否也添加一个与此类似的更改文件？https://github.com/tauri-apps/tauri-action/blob/dev/.changes/fix-tauri-init.md

I have added it

File: .changes/fix-windows-aarch64-bundle.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+action: patch:bug
+---
+
+When building windows-aarch64, looking for x64 bundle.
```
Comment (FabianLars) [inline .changes/fix-windows-aarch64-bundle.md]:
```suggestion
Fix detection of windows arm64 bundles.
```

File: src/build.ts [hunk 2]
```diff
@@ -98,7 +98,15 @@ export async function buildProject(
       join(artifactsPath, `bundle/macos/${fileAppName}.app.tar.gz.sig`),
     ].map((path) => ({ path, arch }));
   } else if (targetInfo.platform === 'windows') {
-    arch = arch.startsWith('i') ? 'x86' : 'x64';
+    // arch = arch.startsWith('i') ? 'x86' : 'x64';
+    // fix windows aarch64 bundle not found
```
Comment (FabianLars) [inline src/build.ts]:
```suggestion
```

**Review by FabianLars — APPROVED** [review]

**Comment by FabianLars** [comment]
thanks again :)

**Pull request merged**